syntax = "proto3";

package chatbot;

service ChatbotService {
    rpc AskChatBot(AskChatBotRequest) returns (AskChatBotResponse);
}

message AskChatBotRequest {
    string question = 1;
    string room_id = 2;
    string organization_id = 3;
    optional string room_key = 4; // NEW: Unique room key for context isolation (use instead of room_id)
}

message AskChatBotResponse {
    string answer = 1;
}
